src/handlers/cell.ts

```diff
@@ -136,6 +135,15 @@ export class CellManager implements IDisposable {
 
     this.previousLineCount = editor.lineCount;
 
+    const editorBreakpoints = this.getBreakpointsInfo(cell);
+    editorBreakpoints.forEach(info => {
+      this.breakpointsModel.addBreakpoint(
+        this._debuggerService.session.client.name,
+        this.getEditorId(),
+        info
+      );
+    });
```

Comment:
I wonder if there should be a method for updating all the breakpoints in a single pass (to avoid sending many breakpoints requests to the kernel). Anyway, this is an optimization that we can do later.

Reply:
Yes it would be better :+1: 

Also thinking about simplifying the `Breakpoints.Model` (there are 2 breakpoint setters at the moment)